app/build.gradle

```diff
@@ -415,13 +402,9 @@ android {
             // next to play store version
 
             // grab commcare version from manifest
-            def manifestParser = new DefaultManifestParser(android.sourceSets.main.manifest.srcFile, new BooleanSupplier() {
-                @Override
-                boolean getAsBoolean() {
-                    return true
-                }
-            }, true, null)
-            def ccVersion = manifestParser.getVersionName()
+            // TODO Fix  No signature of method: build_aul2fcabvs6vrsl6qy1mj4j3h.android() is applicable for argument types: (build_aul2fcabvs6vrsl6qy1mj4j3h$_run_closure7) values: [build_aul2fcabvs6vrsl6qy1mj4j3h$_run_closure7@157fe3a8]
```

Comment:
ping on this

Reply:
Yeah this is something for which I haven't been able to find a replacement for. The documentation for `DefaultManifestParser` doesn't provide any alternative. 
I also haven't seen us use `qabeta` variant in the last 2 years so I'm guessing we can remove it completely?

Reply:
got it, yeah agree that we can remove this variant in another PR.

Reply:
https://github.com/dimagi/commcare-android/pull/2535